server/services/interventionsService.ts

```diff
@@ -157,13 +157,19 @@ export interface ActionPlanAppointment {
   sessionNumber: number
   appointmentTime: string | null
   durationInMinutes: number | null
+  attendance?: AppointmentAttendance
 }
 
 export interface ActionPlanAppointmentUpdate {
   appointmentTime: string
   durationInMinutes: number
 }
 
+interface AppointmentAttendance {
+  attended: string
```

Comment:
Could we restrict this to the allowed values (either with an `enum` or with `"value1" | "value2" | "value3"`)?

Reply:
Have opted for string union type, rather than `enum`, after a bit of googling. It doesn't seem to make a lot of difference either way, but string union means we can write less code as we don't have to import an enum throughout the codebase (among other things).